examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx

```diff
@@ -20,16 +18,12 @@ export default function LivenessDefault({
     stopLiveness,
   } = useLiveness();
 
-  const [error, setError] = useState(undefined);
-  const [checkFailed, setCheckFailed] = useState(false);
```

Comment:
I'm guessing this was never used anywhere?

Reply:
It was previously, but when we removed it we forgot to remove the useState